projects.js

```diff
@@ -0,0 +1,206 @@
+      ${subtitle[1]}
+      </li>
+      <li class="h-2 w-2 bg-light-grey rounded-full"></li>
+      <li class="text-light-grey-4 font-semibold text-small">
+      ${subtitle[2]}
+      </li>
+    </ul>
+    <p class="text-light-teal md:text-base text-[15px]">
+      ${description}
+    </p>
+
+    <ul class="w-[230px] flex justify-between items-center">
+      <li>
+        <a
+          href="#"
+          class="text-light-blue text-xs px-3 py-1 bg-faded-blue rounded-lg"
+          >${technologies[0]}</a
+        >
+      </li>
+      <li>
+        <a
+          href="#"
+          class="text-light-blue text-xs px-3 py-1 bg-faded-blue rounded-lg"
+          >${technologies[1]}</a
+        >
+      </li>
+      <li>
+        <a
+          href="#"
+          class="text-light-blue text-xs px-3 py-1 bg-faded-blue rounded-lg"
+          >${technologies[2]}</a
+        >
+      </li>
+    </ul>
+    <button
+      class="inline-block open-popup-btn transition duration-500 ease-in-out text-primary-blue p-4 text-[17px] border border-light-blue rounded-lg mb-[14px] md:enabled:text-light-blue md:enabled:border-[#7F8CFF] md:enabled:bg-white md:hover:bg-light-blue md:hover:text-white md:active:bg-[#2230D2] md:active:text-white md:disabled:bg-[#e5e5e5] md:disabled:text-[#5E6C84] md:disabled:border-[#C1C7D0]"
+      >${button}</button
+    >
+  </div>
+
+  <!-- modal -->
+  <div class="z-20 hidden modal-wrapper">
+    <div class="modal-content bg-white z-10 rounded-2xl py-[18px] px-4">
+      <header class='flex justify-between items-center'>
+        <h3 class="font-bold text-[32px] self-end">${name}</h3>
+        <button class='close-popup'>
+          <img class="h-[12px] w-[12px]" src='./img/close_3.png' alt='close button'/>
+        </button>
+      </header>
+
+      <!-- modal subtitle -->
+      <div class="text-[13px] font-semibold mt-[22px] flex justify-between items-center w-[75%] md:w-[260px] text-light-grey-4">
+        <span class="uppercase text-[#344563]">${subtitle[0]}</span>
+        <span class="h-[10px] w-[10px] bg-light-grey rounded-full"></span>
+        <span class="capitalize">${subtitle[1]}</span>
+        <span class="h-[10px] w-[10px] rounded-full bg-light-grey"></span>
+        <span>${subtitle[2]}</span>
+      </div>
+
+      <div class="image-wrapper mt-[20px]">
+        <img class="object-cover" src=${featuredImage} alt="project image"/>
+      </div>
+
+      <div class="modal__bottom-section md:flex">
+        <div class="text-wrapper">
+          <p class="mt-[12px] text-[15px] text-[#344563] leading-6">
+            Lorem Ipsum is simply dummy text of the printing and typesetting industry. 
+            Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, 
+            when an unknown printer took a galley of type and scrambled it to make a type 
+            specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent
+          </p>
```

Comment:
Really good job @martinkarugaba. But let's move this also to our `projectsList` array to make it dynamic for the future. In the future, every one of your projects has it is own description so this would be the best practice.